network-api/networkapi/wagtailpages/templates/wagtailpages/blocks/listing_block.html

```diff
@@ -0,0 +1,25 @@
+{% extends "./base_streamfield_block.html" %}
+{% load wagtailcore_tags wagtailimages_tags %}
+
+{% block block_content %}
+	<div class="tw-row tw-py-4">
+		{% for block in self.cards %}
+          {% with card=block count=self.cards|length %}
+            {% image card.image fill-350x197 as img %}
+            <div class="tw-px-4 small:tw-w-full {% if count > 2 %} large:tw-w-1/3 medium:tw-w-1/2 {% else %} medium:tw-w-1/2 {% endif %} tw-mb-5">
+              <div class="card-regular tw-h-full tw-flex tw-flex-col">
+                <img src="{{ img.url }}" alt="{{ card.alt_text }}" />
+                <div class="tw-flex tw-flex-1">
```

Comment:
Note: I've used the alt_text field from the card as per other blocks as there is no alt field on the image model